base/src/main/java/com/mindera/skeletoid/routing/CloseActivityCommand.java

```diff
@@ -8,13 +8,24 @@
 public class CloseActivityCommand implements IRouteCommand {
 
     private final Activity mActivity;
```

Comment:
Maybe convert this to Kotlin and drop hungarian notation? :-)

Reply:
Right away boss! I thought we weren't using kotlin for some reason

Reply:
The only reason is that we started this way back.. when Kotlin wasn't official. 
I think in the mid-term we can even merge base with kotlin extensions modules, and use everything in Kotlin.

:-)